src/sagas.js

```diff
@@ -237,7 +237,8 @@ function* doCheckTokenExpired (action) {
       // No matter saga is canceled for whatever reason, the reload must happen, so here comes the ugly setTimeout()
       setTimeout(() => {
         console.info('======= doCheckTokenExpired() issuing page reload')
-        window.top.location.reload(true)
+        // window.top.location.reload(true)
+        window.location.href = `${AppConfiguration.applicationURL}/`
```

Comment:
Does that actually work? I suspect it might be a problem on dev setup. Snippet from Firefox console:

```
> window.location.href = '//'
[Exception... "The URI is malformed"  nsresult: "0x804b000a (NS_ERROR_MALFORMED_URI)"  location: "JS frame :: debugger eval code :: <TOP_LEVEL> :: line 1"  data: no]
```

 Values of `AppConfiguration.applicationURL`:

* dev setup: `/` (default from `src/config.js`)
* production setup: `/ovirt-engine/web-ui` (from `ovirt-web-ui.config` file)

I guess that property `AppConfiguration.applicationURL` is currently poorly defined, we can agree that it will **always** end with `/` and remove the the slash in line above.